packages/lint-worker/src/helpers.ts

```diff
@@ -7,28 +7,26 @@ import {
 import axios from 'axios';
 import { DbSchema as DbSchemaV1 } from 'languageSupport-next.13';
 
-const oldLinter = '5.20.0';
-
 // for older versions of the language support, the dbschema was not the same,
 // meaning old linters need conversion of the new schema
 export function convertDbSchema(
   originalSchema: DbSchemaV2,
   linterVersion: string,
 ): DbSchemaV2 | DbSchemaV1 {
-  let oldFunctions: Record<string, Neo4jFunction> = {};
-  let oldProcedures: Record<string, Neo4jProcedure> = {};
   if (!originalSchema) {
     return originalSchema;
   }
-  if (
-    originalSchema.functions['CYPHER 5'] &&
-    originalSchema.procedures['CYPHER 5']
-  ) {
-    oldFunctions = originalSchema.functions['CYPHER 5'];
-    oldProcedures = originalSchema.procedures['CYPHER 5'];
-  }
 
-  if (compareMajorMinorVersions(linterVersion, oldLinter) <= 0) {
```

Comment:
This was not correct. The check needs to be any version lower than 2025.01, which is when we changed the schema.